### PR TITLE
Updated logic for PDF responses

### PR DESCRIPTION
**Pull request by arfon**
This is an update to the controller logic for handling PDF responses. In the past we redirected PDF urls but it turns out that the Google Scholar webcrawler doesn't like this. 

With this change, we download the file server-side and send directly to the browser when requesting a PDF, rather than redirecting.

/ cc #130 #637 @labarba

### CODE REVIEW

File: spec/controllers/papers_controller_spec.rb [hunk 1]
```diff
@@ -201,11 +201,20 @@
...
 
-      get :show, params: {doi: paper.doi}, format: "pdf"
-      expect(response).to redirect_to(paper.pdf_url)
+      get :show, params: {doi: paper.doi}
+      expect(response.body).to eq(IO.binread('spec/fixtures/paper.pdf'))
+    end
+
+    it "should not redirect when accepting any content type" do
+      paper = create(:accepted_paper)
+      request.headers["HTTP_ACCEPT"] = "*/*"
```
Comment (arfon) [inline spec/controllers/papers_controller_spec.rb]: This is the content type the Google web crawler specifies.

**Comment by Opussci** [comment]
The inclusion guidelines for the Google Scholar webcrawler state that abstract must be visible on the article page. As I see your journals do not display it.
Maybe that could resolve indexing issue if it is not already resolved.

**Comment by arfon** [comment]
> Maybe that could resolve indexing issue if it is not already resolved.

Thanks @Opussci. Our indexing issues are actually fixed now.

**Comment by Opussci** [comment]
Then sorry for spamming!

**Pull request merged**